src/Microsoft.Health.Fhir.Core/Features/Search/SearchValues/QuantitySearchValue.cs

```diff
@@ -63,7 +64,12 @@ public static QuantitySearchValue Parse(string s)
                 throw new FormatException(Core.Resources.MoreThanTwoTokenSeparatorSpecified);
             }
 
-            decimal quantity = decimal.Parse(parts[0], NumberStyles.Number, CultureInfo.InvariantCulture);
+            decimal quantity;
+            if (decimal.TryParse(parts[0], NumberStyles.Number, CultureInfo.InvariantCulture, out quantity) == false)
```

Comment:
Elsewhere in the code, we generally use the `!` operator when testing for false on `TryParse`.  
```suggestion
            if (!decimal.TryParse(parts[0], NumberStyles.Number, CultureInfo.InvariantCulture, out quantity))
```